### PR TITLE
Proj Speed Up

### PR DESCRIPTION
**Pull request by acpaquette**
Moves proj CRS creation into camera instantiation and properly cleans up proj objects once finished.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: tests/ProjectedCameraTests.cpp [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+#include <iostream>
+
+using json = nlohmann::json;
+
+TEST_F(ConstVelocityProjectedSensorModel, ConstructFromState) {
+  std::string modelState = sensorModel->getModelState();
+  csm::RasterGM * model = getUsgsCsmModelFromState(modelState, UsgsAstroProjectedSensorModel::_SENSOR_MODEL_NAME, NULL);
+
+  // When this is different, the output is very hard to parse
+  // TODO implement JSON diff for gtest
```
Comment (AustinSanders) [inline tests/ProjectedCameraTests.cpp:21]: Is this something that needs to be done before merge, or just a QOL thing for debugging?

Comment (acpaquette) [reply tests/ProjectedCameraTests.cpp:21]: This was a comment that was ripped from the line scan camera test that this was copied from. I can remove it just so we don't duplicate it

**Review by AustinSanders — APPROVED** [review]

**Review by chkim-usgs — APPROVED** [review]
Ran the tests locally with `PROJ_IGNORE_CELESTIAL_BODY=Yes`, tests pass. Looks good.
